Normalize.lua

```diff
@@ -103,3 +128,18 @@ function Normalize:__tostring__()
   end
   return string.format(s,torch.type(self),self.p)
 end
+
+function Normalize:type(type)
```

Comment:
should implement the tensorCache argument (https://github.com/torch/nn/blob/master/Module.lua#L116)